### DISCUSSION TITLE
Sui Stake Protocol

### DISCUSSION THREAD

**Discussion by hotboice** [NonceGeek/Move-Camp#230] [Amazing Things!]
简单描述业务流程: 
1、用户注入代币建立池子
2、质押指定的NFT, 根据地板价和质押率, 得到可借款金额
3、从池子借出代币
4、每一笔借、还的操作都会获得平台币奖励
5、清算机制
更详细的需求和业务流程可加入之后探讨
现阶段3-4人的团队较为合适, 目前缺少web3前端经验和一位Sui Move开发者
期待加入, 一起交流和学习Sui 🌹🌹🌹

**Comment by v1xingyue** [comment]
看热闹的。

**Comment by xsstc** [comment]
还缺合约的人吗  有每天的事件限制吗

**Reply by hotboice** [reply]
还缺，每天大概5小时左右

**Comment by xiegudong45** [comment]
想问下现在还缺人吗？

**Comment by v1xingyue** [comment]
给贴个 github 链接么？ 想学习一下。

**Comment by zengxinhai** [comment]
repo：https://github.com/zengxinhai/sui-stake-protocol
